keyboards/handwired/split_cloud/info.json

```
@@ -0,0 +1,120 @@
+{
+    "keyboard_name": "Split-Cloud",
+    "manufacturer": "coolsa",
+    "maintainer": "Cloud Chagnon",
+    "url": "https://github.com/coolsa/cloud_keyboard",
+    "usb": {
+        "vid": "0xCC43",
+        "pid": "0x0001",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "cols": ["B6", "B2", "B3", "B1", "F7", "F6"],
+        "rows": ["C6", "D7", "E6", "B4", "B5"]
+    },
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 0]
```

default matrix co-ordinates for bootmagic
```suggestion
        "enabled": true
```